Extension/src/LanguageServer/client.ts

```diff
@@ -1194,6 +1198,9 @@ export class DefaultClient implements Client {
                 files: {
                     encoding: settings_filesEncoding
                 },
+                editor: {
```

Comment:
If you choose to keep tabSize (which is not longer read on the native side), I think it would need to be added to this editor node, to match the change you made in sendAllSettings().  Though, I think it would be better to remove tabSize entirely.  (Also, the set_tab_size() declaration can be removed from settings.h on the native side, as it's not defined).